apps/studio/components/interfaces/TableGridEditor/TableGridEditor.tsx

```diff
@@ -210,7 +211,12 @@ const TableGridEditor = ({
    */
 
   return (
-    <>
+    <div
```

Comment:
bit crap, but when any click happens in a table tab, the tab becomes permanent.
bit hacky